cowprotocol/cowswap
# Fix pending state for limit orders

# Summary

Fixes #1488 

This PR will remove pending indicator from the header status component when there are open limit orders and will only show pending indicator for the open swap orders.

## To test
- create a limit order
- there should be no pending indicator in the header status field
- create a swap order
- there should be pending indicator as usual
